Source/WebCore/platform/ScreenProperties.h

```diff
@@ -49,11 +49,11 @@ struct ScreenData {
     uint32_t displayMask { 0 };
     PlatformGPUID gpuID { 0 };
     DynamicRangeMode preferredDynamicRangeMode { DynamicRangeMode::Standard };
-    WEBCORE_EXPORT double screenDPI() const;
+    WEBCORE_EXPORT double monitorDPI() const;
```

Comment:
I'm fine renaming screenDPI with fontDPI when appropriate, but we can leave screenDPI to refer to the monitorDPI, since it's used by other ports and part of ScreenProperties after all.

Reply:
Alright, all occurrences of monitorDPI will go back to what they were before.